apps/web/src/state/akka/actions.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from '@reduxjs/toolkit'
```

Comment:
let's not add a custom akka folder but simply add a quoter field to the quotes/trades as suggested above. Keep it universal.